### ISSUE TITLE
Rule proposal: enforce selector result name

### ISSUE THREAD

**Issue by EvgenyOrekhov** [DianaSuvorova/eslint-plugin-react-redux#98]
Enforce `const <name> = useSelector(select<Name>)` naming convention.

It will make it easier to see which values and selector functions are related, and it will make it easier to grep code.

Examples of **incorrect** code:

```js
const foo = useSelector(selectBar); // Should warn to use `const bar`
const posts = useSelector(selectUserPosts); // Should warn to use `const userPosts`
```

Examples of **correct** code:

```js
const bar = useSelector(selectBar);
const userPosts = useSelector(selectUserPosts);
const { foo } = useSelector(selectBar); // Destructuring is not checked
const foo = useSelector((state) => state.bar); // Inline functions are not checked
```
